packages/invoices/src/InboundShipment/ListView/Toolbar.tsx

```diff
@@ -1 +1,83 @@
-export const Toolbar = () => null;
+import React, { FC, useEffect, useRef } from 'react';
+import {
+  useNotification,
```

Comment:
more copy paste.

I'm feeling shitty about it but I also don't want to bundle all this together yet - I have a feeling I will regret it.. 🤷